Avaliação dos desafios iniciais (#1)
betrybe/sd-0x-mysql-all-for-one: README.md

# Bem vindo ao repositório do projeto All For One

Você já usa o GitHub diariamente para desenvolver os exercícios, certo? Agora, para desenvolver os projetos, você deverá seguir as instruções a seguir. Fique atento a cada passo e, se tiver qualquer dúvida, nos envie por Slack! #vqv 🚀

Aqui você vai encontrar os detalhes de como estruturar o desenvolvimento do seu projeto a partir desse repositório, utilizando uma branch específica e um Pull Request para colocar seus códigos.

---

### ANTES DE COMEÇAR A DESENVOLVER:

1. Clone o repositório

>   * `git clone https://github.com/tryber/sd-02-block21-mysql-all-for-one`.

Reviewer: ```suggestion
  * `git clone https://github.com/tryber/sd-0x-mysql-all-for-one`.
```

  * Entre na pasta do repositório que você acabou de clonar:

>     * `cd sd-02-block21-mysql-all-for-one`

Reviewer: ```suggestion
    * `cd sd-0x-mysql-all-for-one`
```

2. Crie uma branch a partir da branch `master`
  * Verifique que você está na branch `master`
    * Exemplo: `git branch`
  * Se não estiver, mude para a branch `master`
    * Exemplo: `git checkout master`
  * Agora, crie uma branch onde você vai guardar os `commits` do seu projeto
    * Você deve criar uma branch no seguinte formato: `nome-de-usuario-nome-do-projeto`

>     * Exemplo: `git checkout -b jhonatan-de-souza-mysql-all-for-one`

Reviewer: ```suggestion
    * Exemplo: `git checkout -b joaozinho-mysql-all-for-one`
```

3. Para cada exercício você deve criar um novo arquivo sql seguindo a seguinte estrutura:
  * desafio1.sql, desafio2.sql...desafioN.sql

4. Adicione as mudanças ao _stage_ do Git e faça um `commit`
  * Verifique que as mudanças ainda não estão no _stage_
    * Exemplo: `git status` (deve aparecer o arquivo que você alterou como desafio1.sql)
  * Adicione o novo arquivo ao _stage_ do Git
      * Exemplo:
        * `git add .` (adicionando solução para desafio 1)
        * `git status` (deve aparecer listado o arquivo _desafio1.sql_ em verde)
  * Faça o `commit` inicial
      * Exemplo:
        * `git commit -m 'iniciando o projeto MySQL All For One'` (fazendo o primeiro commit)
        * `git status` (deve aparecer uma mensagem tipo _nothing to commit_ )

6. Adicione a sua branch com o novo `commit` ao repositório remoto
  * Usando o exemplo anterior: `git push -u origin seunome-mysql-all-for-one`

7. Crie um novo `Pull Request` _(PR)_

>   * Vá até a página de _Pull Requests_ do [repositório no GitHub](https://github.com/tryber/sd-02-block21-mysql-all-for-one/pulls)

Reviewer: ```suggestion
  * Vá até a página de _Pull Requests_ do [repositório no GitHub](https://github.com/tryber/sd-0x-mysql-all-for-one/pulls)
```

  * Clique no botão verde _"New pull request"_
  * Clique na caixa de seleção _"Compare"_ e escolha a sua branch **com atenção**
  * Clique no botão verde _"Create pull request"_
  * Adicione uma descrição para o _Pull Request_ e clique no botão verde _"Create pull request"_
  * **Não se preocupe em preencher mais nada por enquanto!**

>   * Volte até a [página de _Pull Requests_ do repositório](https://github.com/tryber/sd-02-block21-mysql-all-for-one/pulls) e confira que o seu _Pull Request_ está criado

Reviewer: ```suggestion
  * Volte até a [página de _Pull Requests_ do repositório](https://github.com/tryber/sd-0x-mysql-all-for-one/pulls) e confira que o seu _Pull Request_ está criado
```

22. Atualize os dados de `discount` do `order_details` para 15.
23. Atualize os dados de `discount` da tabela `order_details` para 30 cuja `unit_price` seja menor que 10.0000.
24. Atualize os dados de discount da tabela `order_details` para 45 cuja `unit_price` seja maior que 10.0000 e o id seja um número entre 30 a 40.
25. Delete todos os dados em que a `unit_price` da tabela `order_details` seja menor que 10.0000.
26. Delete todos os dados em que a `unit_price` da tabela `order_details` seja maior que 10.0000.
27. Delete todos os dados da tabela `order_details`.

---

## Instruções para entregar seu projeto:

### DEPOIS DE TERMINAR O DESENVOLVIMENTO

Para **"entregar"** seu projeto, siga os passos a seguir:

* Vá até a página **DO SEU** _Pull Request_, adicione a label de _"code-review"_ e marque seus colegas
  * No menu à direita, clique no _link_ **"Labels"** e escolha a _label_ **code-review**
  * No menu à direita, clique no _link_ **"Assignees"** e escolha **o seu usuário**

>   * No menu à direita, clique no _link_ **"Reviewers"** e digite `students`, selecione o time `tryber/students-sd-02`

Reviewer: ```suggestion
  * No menu à direita, clique no _link_ **"Reviewers"** e digite `students`, selecione o time `tryber/students-sd-0x`
```
